funcions/2.js

```diff
@@ -0,0 +1,18 @@
+function oldSum(list) {
+	var finalValue = 0
+	for (var i = 0; i < list.length; i++) {
+		finalValue += list[i]
+	}
+	return finalValue
+}
+
+function newSum(list) {
+	const reducer = (acc, cur) => acc + cur
+	const finalVal = list.reduce(reducer)
```

Comment:
I'd avoid storing iteration function to separate variable. Consider a line of code (or an expression) as a line of thought. 

```
list.reduce((acc, l) => acc + l, 0);
```
Would read -- reduce the list by adding every item.

vs

```
list.reduce(reducer);
```

Would read -- reduce the list with reducer. There's not enough contextual info in the line itself. You'd have to search around the codebase to find out more. Makes the code a bit harder to read.